esx_service/utils/error_code.py

```diff
@@ -18,6 +18,8 @@
 VM_NOT_BELONG_TO_TENANT = "VM {0} does not belong to any tenant"
 TENANT_NOT_EXIST = "Tenant {0} does not exist"
 TENANT_ALREADY_EXIST = "Tenant {0} already exists"
+TENANT_CREATE_FAILED = "Tenant {0} create failed with err:{1}"
+TENANT_SET_ACCESS_PRIVILEGES_FAILED = "Tenant {0} set access privileges on datastore {1} failed with {2}"
```

Comment:
Nit: Let's keep the error messages consistent: either "failed with {}" or "failed with error: {}"